go.mod

```diff
@@ -1,22 +1,32 @@
-module go.amzn.com/sagemaker/sagemaker-k8s-operator
+module github.com/aws/amazon-sagemaker-operator-for-k8s
 
 go 1.12
 
 require (
+	cloud.google.com/go v0.38.0 // indirect
 	github.com/Jeffail/gabs/v2 v2.0.0
 	github.com/adammck/venv v0.0.0-20160819025605-8a9c907a37d3
 	github.com/aws/aws-sdk-go v1.25.36
 	github.com/aws/aws-sdk-go-v2 v0.11.0
 	github.com/go-logr/logr v0.1.0
+	github.com/google/btree v1.0.0 // indirect
 	github.com/google/go-cmp v0.3.1
 	github.com/google/uuid v1.1.1
+	github.com/gregjones/httpcache v0.0.0-20190611155906-901d90724c79 // indirect
 	github.com/nsf/jsondiff v0.0.0-20190712045011-8443391ee9b6
-	github.com/onsi/ginkgo v1.8.0
-	github.com/onsi/gomega v1.5.0
+	github.com/onsi/ginkgo v1.10.1
+	github.com/onsi/gomega v1.7.0
+	github.com/peterbourgon/diskv v2.0.1+incompatible // indirect
 	github.com/pkg/errors v0.8.1
-	golang.org/x/net v0.0.0-20190501004415-9ce7a6920f09
-	k8s.io/apimachinery v0.0.0-20190404173353-6a84e37a896d
-	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
+	golang.org/x/crypto v0.0.0-20190820162420-60c769a6c586 // indirect
+	golang.org/x/net v0.0.0-20191004110552-13f9640d40b9
+	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45 // indirect
+	golang.org/x/time v0.0.0-20190308202827-9d24e82272b4 // indirect
+	google.golang.org/appengine v1.5.0 // indirect
+	k8s.io/apimachinery v0.0.0-20190816221834-a9f1d8a9c101
+	k8s.io/cli-runtime v0.0.0-20190918224932-e56234cc6b3d // indirect
+	k8s.io/client-go v11.0.1-0.20190918222721-c0e3722d5cf0+incompatible
+	k8s.io/utils v0.0.0-20191114184206-e782cd3c129f // indirect
```

Comment:
It picked up automatically ?

Reply:
Correct, I did not manually override any modules.